File: src/main/java/br/com/matheuspadilha/tasks/error/FieldMessage.java
package br.com.matheuspadilha.tasks.error;

import com.fasterxml.jackson.annotation.JsonInclude;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serial;
import java.io.Serializable;

import static com.fasterxml.jackson.annotation.JsonInclude.Include;

@Data
@AllArgsConstructor
@NoArgsConstructor
@JsonInclude(Include.NON_NULL)
public class FieldMessage implements Serializable {

    @Serial
    private static final long serialVersionUID = 1L;

    private String fieldName;
    private String message;

}

File: src/main/kotlin/br/com/matheuspadilha/tasks/error/FieldMessage.kt
package br.com.matheuspadilha.tasks.error

import com.fasterxml.jackson.annotation.JsonInclude
import com.fasterxml.jackson.annotation.JsonInclude.Include
import java.io.Serializable

@JsonInclude(Include.NON_NULL)
data class FieldMessage(
    val fieldName: String? = null,
    val message: String? = null
) : Serializable